miso-dto/src/main/java/uk/ac/bbsrc/tgac/miso/dto/Dtos.java

```diff
@@ -1477,6 +1477,10 @@ public static PoolDto asDto(Pool from, boolean includeContents) {
   }
 
   public static RunDto asDto(Run from) {
+    return asDto(from, false, false, false);
+  }
+
+  public static RunDto asDto(Run from, boolean includeContainers, boolean includeContainerPartitions, boolean includePoolContents) {
```

Comment:
I'd much rather this be include everything or nothing. Is there a case where you want only part of the hierarchy?

Reply:
I thought I'd leave it more configurable rather than less, but sure, this could be a single toggle.